scripts/intfutil

```diff
@@ -38,7 +38,7 @@ def appl_db_keys_get(appl_db, intf_name):
     if intf_name is None:
         appl_db_keys = appl_db.keys(appl_db.APPL_DB, "PORT_TABLE:*")
     elif intf_name.startswith('Ethernet'):
-        appl_db_keys = db.keys(appl_db.APPL_DB, "PORT_TABLE:%s" % intf_name)
+        appl_db_keys = appl_db.keys(appl_db.APPL_DB, "PORT_TABLE:%s" % intf_name)
```

Comment:
If this change alone fixes a bug, please submit this change in a PR by itself.

Reply:
Hi Joe, thanks for the review. For this specific issue, how do i submit in a different PR.
I am new to github.
I tried to push this change from the forked repo, thinking that I can submit a different pull request for this change. But this fix showed up automatically in this pull request 424 itself.
What do i need to do, to push the changes in the same forked repo, so as to show up as a new pull request.

Reply:
I suggest you create a new branch in your forked repo. In that branch, make only the change for the bugfix, then commit that change and push your changes to your remote fork. Then come back here and create a new pull request from your new branch to the master branch of this repo.

Reply:
Thanks Joe. I revoked the changes in intfutil from this pull request.
I resubmitted the changes for #424 (main.py) as suggested by you. Once these changes get approved and merged to master, i will raise a pull request on a topic branch for the other PR off the forked repo.
Can you please review the changes re-submitted for this issue?

Reply:
Hi Joe, do you have any more review comments in config/main.py?
Can this be PR please be merged?

Reply:
@kirankella: Since you removed this change and will be submitting it separately, this PR no longer "fixes (SONiC/issues/268)", correct? If not, please remove that from the PR title.

Reply:
No. This PR fixes SONiC/issues/268 only. The removed change fixes https://github.com/Azure/sonic-utilities/issues/427.

Reply:
Review comments are addressed.